packages/css/src/less/select.less

```
@@ -29,10 +29,25 @@
     }
   }
 
+  .dropdown-toggle.btn-input {
+    &.btn-sm {
+      padding-right: calc(var(--space-8) * 4);
```

No need to do the calc() as we have spacing tokens.

```suggestion
      padding-right: var(--space-32);
```

---

thanks @anttio